src/components/pages/404-pages/VuesticPageNotFoundLargeText.vue

```diff
@@ -1,28 +1,26 @@
 <template>
   <div class="vuestic-page-not-found-large-text">
     <div class="vuestic-page-not-found-large-text__content">
-      <div class="vuestic-page-not-found-large-text__wallpaper">
-        <router-link
-          class="i-vuestic vuestic-page-not-found-large-text__i-vuestic"
-          :to="{path: '/'}"/>
+      <div class="vuestic-page-not-found-large-text__wallpaper d-flex justify--center
+        align--start overflow--hidden">
+        <h1 class="vuestic-page-not-found-large-text__number mb-0">404</h1>
       </div>
-      <div class="vuestic-page-not-found-large-text__wallpaper">
-        <h1 class="vuestic-page-not-found-large-text__number">404</h1>
-      </div>
-      <div class="vuestic-page-not-found-large-text__wallpaper">
+      <div class="vuestic-page-not-found-large-text__wallpaper d-flex justify--center
+        align--start overflow--hidden">
         <h3 class="vuestic-page-not-found-large-text__text">The page's gone
           fishing.</h3>
       </div>
-      <div class="vuestic-page-not-found-large-text__wallpaper flex md12">
+      <div class="vuestic-page-not-found-large-text__wallpaper d-flex justify--center
+        align--start overflow--hidden flex md12">
         <div class="va-row vuestic-page-not-found-large-text__message justify--center">
-          <h4 class="vuestic-page-not-found-large-text__text-small">
+          <span class="vuestic-page-not-found-large-text__text-small mt-4">
             If you feel that it's not right, please send us a message at &nbsp;
-          </h4>
-          <a href="mailto:hello@epicmax.co">hello@epicmax.co</a>
+          </span>
+          <a class="mt-4" href="mailto:hello@epicmax.co">hello@epicmax.co</a>
         </div>
       </div>
-      <div
-        class="vuestic-page-not-found-large-text__wallpaper-button">
+      <div class="vuestic-page-not-found-large-text__wallpaper d-flex
+        justify--center align--start pt-4 pb-3 overflow--hidden">
         <va-button :to="{ name: 'dashboard' }">
```

Comment:
Button not clickable.

![image](https://user-images.githubusercontent.com/5394573/53020435-ed5bc100-3467-11e9-8bd6-bc970cea7ae8.png)

Reply:
Not reproduced